inc/commonitiltask.class.php

```diff
@@ -549,11 +549,11 @@ function cleanDBonPurge() {
    // SPECIFIC FUNCTIONS
 
    /**
```

Comment:
You should remove the entire PHPDoc block, as it provides no usefull information.

Same remark for other classes except `CommonDBTM`.